x-pack/plugins/observability/public/index.ts

```diff
@@ -118,6 +122,7 @@ export {
 } from './components/shared/exploratory_view/configurations/constants';
 export { ExploratoryViewContextProvider } from './components/shared/exploratory_view/contexts/exploratory_view_config';
 export { fromQuery, toQuery } from './utils/url';
+export { getAlertSummaryTimeRange } from './utils/alert_summary_widget';
```

Comment:
It would be better to move this utility to a package and then use it in both plugins, @XavierM, which package do you think would be better for this alert utility?

Reply:
Creating a tech debt ticket for this sounds good to me

Reply:
@maryam-saeidi here we have it https://github.com/elastic/kibana/issues/151593

Reply:
Thanks! 🙏🏻